cmd/ttnv3/ttnv3.go

```diff
@@ -0,0 +1,22 @@
+// Copyright © 2023 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ttnv3
+
+import "go.thethings.network/lorawan-stack-migrate/pkg/commands"
+
+const sourceName = "ttnv3"
+
+// TTNv3Cmd represents the ttnv3 source.
+var TTNv3Cmd = commands.Source(sourceName, "Export devices from The Things Stack")
```

Comment:
Is it too late to change `ttnv3` to `tts`?

Reply:
I don't think it's too late since there were only release candidates with the `ttnv3`. We could change name to `tts` and add `ttnv3` alias.